modules/ROOT/pages/access-logging.adoc

```diff
@@ -96,15 +96,22 @@ The following table lists the available log format options.
 |%o or %{HeaderName}o
 |`HeaderName` header value from the response
 
+|%p
+|The canonical port of the server serving the request
```

Comment:
The canonical port of the server serving the request
-->
The canonical port of the server that serves the request